docs/tutorials/tabular_prediction/tabular-custom-model.md

```diff
@@ -0,0 +1,374 @@
+# Adding a custom model to AutoGluon
+:label:`sec_tabularcustommodel`
+
+**Tip**: If you are new to AutoGluon, review :ref:`sec_tabularquick` to learn the basics of the AutoGluon API.
+
+This tutorial describes how to add a custom model to AutoGluon that can be trained and ensembled alongside the default models ([default model documentation](../../api/autogluon.tabular.models.html#module-autogluon.tabular.models)).
```

Comment:
trained, hyperparameter-tuned, and ensembled

Reply:
added